src/fiskaltrust.Middleware.Interface.Client.Http/POS/HttpPos.cs

```diff
@@ -195,10 +222,20 @@ Stream ifPOS.v0.IPOS.Journal(long ftJournalType, long from, long to)
         {
             using (var client = GetClient(_options))
             {
+                string format;

```

Comment:
Journal is always json and doesn't support XML, we need to roll this back I think 😅